Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debian:jessie
```

Comment:
rust/llvm does not compile o  alpine which uses musl instead of the standard libc. I tried avarything I could to make it work but always ended up with errors :/